dotCMS/src/main/java/com/dotcms/rest/ContentResource.java

```diff
@@ -634,106 +638,120 @@ public Object unmarshal(HierarchicalStreamReader reader, UnmarshallingContext co
     @Path("/{params:.*}")
     @Produces(MediaType.TEXT_PLAIN)
     @Consumes(MediaType.MULTIPART_FORM_DATA)
-    public Response multipartPUT(@Context HttpServletRequest request, @Context HttpServletResponse response,
-            FormDataMultiPart multipart,@PathParam("params") String params) 
-                    throws URISyntaxException, DotDataException, DotSecurityException {
+    public Response multipartPUT(@Context HttpServletRequest request,
+            @Context HttpServletResponse response,
+            FormDataMultiPart multipart, @PathParam("params") String params)
+            throws URISyntaxException, DotDataException, DotSecurityException {
         return multipartPUTandPOST(request, response, multipart, params, "PUT");
     }
-    
+
     @POST
     @Path("/{params:.*}")
     @Produces(MediaType.TEXT_PLAIN)
     @Consumes(MediaType.MULTIPART_FORM_DATA)
-    public Response multipartPOST(@Context HttpServletRequest request, @Context HttpServletResponse response,
-            FormDataMultiPart multipart,@PathParam("params") String params) throws URISyntaxException, DotDataException, DotSecurityException {
+    public Response multipartPOST(@Context HttpServletRequest request,
+            @Context HttpServletResponse response,
+            FormDataMultiPart multipart, @PathParam("params") String params)
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove this useless assignment to local variable "user". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1854)
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Remove this unused "user" local variable. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS1481)